src/functional/fencil_processors/codegens/gtfn/gtfn_module.py

```diff
@@ -30,8 +30,21 @@ def create_source_module(
     """Generate GTFN C++ code from the ITIR definition."""
     function = source_modules.Function(itir.id, parameters)
 
-    rendered_params = ", ".join(["gridtools::fn::backend::naive{}", *(p.name for p in parameters)])
-    decl_body = f"return generated::{function.name}(nullptr)({rendered_params});"
+    rendered_params = ", ".join(
+        [
+            "gridtools::fn::backend::naive{}",
+            *[
+                p.name
+                for p in parameters
+                if not isinstance(p, source_modules.ConnectivityParameter)
+            ],
+        ]
+    )
+    conn_params = [
+        p.name for p in parameters if isinstance(p, source_modules.ConnectivityParameter)
+    ]
+    rendered_connectivity = f"gridtools::hymap::keys<{', '.join(f'generated::{p}_t' for p in conn_params)}>::make_values({', '.join(f'gridtools::fn::as_neighbor_table<int,4>({c})' for c in conn_params)})"  # TODO std::forward, type and max_neighbors
```

Comment:
With Peters PR, we need to wrap the python buffer in a sid, then wrap the result in a neighbor_table, see https://github.com/GridTools/gridtools/pull/1730

Reply:
Implemented in: https://github.com/GridTools/gt4py/pull/939